hackathon_site/event/jinja2/event/form_base.html

```diff
@@ -14,7 +14,7 @@
             <span class="helper-text">{{ field.help_text|safe }}</span>
         {% endif %}
         {% if field.errors %}
-            <span class="formFieldError">
+            <span class="formFieldError col s12">
```

Comment:
This is a col inside of a col without a row (and whatever you do to this has to be done to the `render_checkbox_field` macro below as well)